src/cards/freeze.ts

```diff
@@ -104,6 +104,7 @@ const spell: Spell = {
           }
         }
       }
+      return false;
```

Comment:
I think returning a boolean for onTurnEnd shouldn't be necessary, the boolean for onTurnStart is only needed for skipping the unit's turn.  For example, freeze returns true so that that unit wont take a turn while they're frozen